### ISSUE TITLE
Using System.Numerics.BigInteger breaks IL2CPU: Multiple Methods definitions.

### ISSUE THREAD

**Issue by Guillermo-Santos** [CosmosOS/IL2CPU#206]
As the tittle says, i was trying to use a variable of type `BigInteger`:
``` csharp
var bi = new BigInteger(0);
Console.WriteLine($"BigInt: {bi}");
```

 but got the next output on build:
[buildOut.txt](https://github.com/CosmosOS/IL2CPU/files/12643896/buildOut.txt)

**Comment by Guillermo-Santos** [comment]
Side note, the error does not contain any mention of the need of a Plug.

**Comment by Guillermo-Santos** [comment]
After reading a little, I think it do have the same cause.

**Comment by zarlo** [comment]
this should be fix by #209 will need to check

**Comment by Guillermo-Santos** [comment]
Yes, it fixed it. Now an easy Fibonacci Series implementation is possible xd.
